Import documentation for CGUs-versions and CGUs-snapshots (#81)
ambanum/CGUs: README.md

Néanmoins, le **bruit** peut changer de forme avec le temps. Afin d'éviter des pertes d'information irrécupérables pendant l'étape de **filtrage du bruit**, un **instantané** de la page Web est **enregistré** à chaque **évolution**. Après avoir **filtré l'instantané** de son **bruit**, si le **document** résultant a changé par rapport à sa **version** précédente, une nouvelle **version** est **enregistrée**.

Vous pouvez disposer de votre propre instance **privée** de l'outil _CGUs_ et suivre vous-même les **évolutions**. Néanmoins, nous **publions** chaque **version** sur une [instance **publique**](https://github.com/ambanum/CGUs-versions) facilitant l'exploration de l'**historique** et **notifiant** par courriels l'**enregistrement** de nouvelles **versions**. Les **utilisateurs** peuvent [**s'abonner** aux **notifications**](#recevoir-des-notifications).

_Note: Actuellement, nous privilégions les **conditions** rédigées en anglais et concernant la juridiction européenne._

> ## [CGUs-Versions](https://github.com/ambanum/CGUs-versions)

Reviewer: ```suggestion
## Exploring the history of terms of service
```

> Database of versions recorded each time there is a change in the Terms of Service of the main online service.

Reviewer: ```suggestion
We offer a public database of versions recorded each time there is a change in the terms of service and other contractual documents of tracked services: [CGUs-Versions](https://github.com/ambanum/CGUs-versions).
```

> > Base de données des versions enregistrés à chaque évolution des Conditions Générales d'Utilisations des principaux services en ligne.

Reviewer: ```suggestion
> Nous mettons à disposition une base de données des versions enregistrées à chaque évolution des conditions générales d'utilisation et des autres documents contractuels des services en ligne suivis : [CGUs-Versions](https://github.com/ambanum/CGUs-versions).
```

- Le second, appelé *rich diff* (bouton avec l'icône document) permet d'**unifier l'ensemble des modifications sur un seul document** (pour notre [exemple](https://github.com/ambanum/CGUs-versions/commit/58a1d2ae4187a3260ac58f3f3c7dcd3aeacaebcd?short_path=e8bdae8#diff-e8bdae8692561f60aeac9d27a55e84fc)). La couleur **rouge** montre les éléments **supprimés**, la couleur **jaune** montre les paragraphes **modifiés**, et la couleur **verte** montrent les éléments **ajoutés**. Attention, cet affichage **ne montre pas certaines modifications** comme le changement des hyperliens et le style du texte.

**Remarques** :

- Pour les longs documents, les **paragraphes inchangés ne seront pas affichés par défaut**. Vous pouvez manuellement les faire apparaître en cliquant sur les petites flèches juste au-dessus ou juste en-dessous des paragraphes affichés.
- Vous pouvez utiliser le bouton **History n'importe où** dans le dépôt CGUs-versions, qui affichera alors l'**historique des modifications subies par tous les documents se trouvant dans le dossier** où vous vous trouvez (y compris dans les sous-dossiers).

## [CGUs-Snapshots](https://github.com/ambanum/CGUs-snapshots)

> Database of snapshots recorded each time there is a change in the Terms of Service of the main online service.

Reviewer: ```suggestion
Database of snapshots recorded each time there is a change in the terms of service and other contractual documents of tracked services.
```

> Base de données des instantanés enregistrés à chaque évolution des Conditions Générales d'Utilisations des principaux services en ligne.

- - -

> ## Developement

Reviewer: ```suggestion
## Development
```
